### PR TITLE
Add `spotlights`, `vscode-plugin` and `marimo-tutorials` repositories

### PR DESCRIPTION
**Pull request by Haleshot**
Add:
1. Spotlights repo
2. marimo-tutorials repo in respective categories
3. Add talks/meetings relating to marimo as found on socials (discord).
4. Hyperlink more relevant data pertaining to the talks
5. Courses (calmcode) and NASA project/course and Stanford Lab's project

### CODE REVIEW

**Comment by Haleshot** [comment]
Edit: Ok, didn't see the recent commits made by @akshayka 😅; will revisit this PR and commits soon. Getting a bit late here.

File: README.md [hunk 1]
```diff
@@ -54,8 +57,12 @@ _Add your tutorial here!_
...
+- [CVXPY NASA Course](https://github.com/marimo-team/spotlights/tree/main/005-cvxpy-nasa) - Interactive convex optimization course for NASA scientists using marimo notebooks.
 
 ## Talks
 
-- [marimo: an open-source reactive notebook for Python](https://www.youtube.com/watch?v=9R2cQygaoxQ) - North Bay Python, 2024.
+- [marimo: an open-source reactive notebook for Python](https://www.youtube.com/watch?v=9R2cQygaoxQ) - *North Bay Python, 2024* ([slides](https://docs.google.com/presentation/d/1dzTuViK7sn4LKLod-7aR828FI3-hBEdxZLDNdL9L-rI/edit#slide=id.g2d05ea9da64_0_536), [conference info](https://pretalx.northbaypython.org/nbpy-2024/talk/LSLE9A/))
+- [Lessons learned from building a reproducible notebook for AI/ML using graphs](https://www.youtube.com/live/3e9YAdOpihI?si=3cMlUJoFoIVDnmxx&t=1738) - Technical talk at MLOps Learners community
+- [How to rethink the notebook](https://youtu.be/srQ3ESat6u4) - Interview with Akshay Agrawal discussing marimo's development and features
+- [Community Call #1](https://drive.google.com/file/d/1_3cScwQLR_Iz8wbScv6rzP7C2CZDR_tb/view?usp=sharing) - First marimo community call discussing updates and developments
```
Comment (mscolnick) [inline README.md:68]: I don't think this is a useful resource - we can remove this

**Comment by mscolnick** [comment]
@Haleshot no rush - I'm going to push up a change too to fix linting for the recent commit

**Review by Haleshot — COMMENTED** [review]
Added relevant repos (as discussed in discord too). Looks good?

**Review by mscolnick — APPROVED** [review]

**Pull request merged**